datarepo-clienttests/src/main/java/utils/FileUtils.java

```diff
@@ -53,4 +60,51 @@ public static InputStream getJSONFileHandle(String resourceFilePath)
     String[] resourceFileNames = resourceDirectoryFile.list();
     return resourceFileNames == null ? new ArrayList<>() : Arrays.asList(resourceFileNames);
   }
+
+  /**
+   * Fetch the gsPath path.
+   *
+   * <p>// take the file name (e.g. testRetrieveDRSSnapshot.json) and byte array // build the
+   * blobinfo and create the storage object // save a reference to the blobid (or whatever you need
+   * to delete the storage object) in some static list in that class
+   *
+   * @param fileId the if for the fileModel
+   * @param storage the storage
+   * @param testConfigGetIngestbucket the bucket
+   * @return the gsPath
+   */
+  public static String getFileRefs(
+      String fileId, Storage storage, String testConfigGetIngestbucket) {
+    createdScratchFiles = new ArrayList<>();
+    // generate a JSON file with the fileref
+    String jsonLine =
+        "{\"VCF_File_Name\":\"name1\", \"Description\":\"description1\", \"VCF_File_Ref\":\""
+            + fileId
+            + "\"}\n";
```

Comment:
Could you modify this method to:
1) take the result of jsonLine.getBytes as a parameter instead of building the line here?
2) take the dirInCloud as a parameter instead of hard-coding it here?
That way, this method could be used by other datasets, possibly with different schemas, and the name won't always collide.